dtfs-central-api/src/v1/controllers/utilisation-report-service/get-selected-fee-records-details.controller/helpers.ts

```diff
@@ -74,8 +74,11 @@ export const canFeeRecordsBeAddedToExistingPayment = async (reportId: string, fe
     return false;
   }
 
-  const anyFeeRecordDoesNotHaveStatusToDo = feeRecords.some((record) => record.status !== FEE_RECORD_STATUS.TO_DO);
-  if (anyFeeRecordDoesNotHaveStatusToDo) {
+  const allFeeRecordsHaveToDoStatus = feeRecords.every(
+    (record) => record.status === FEE_RECORD_STATUS.TO_DO || record.status === FEE_RECORD_STATUS.TO_DO_AMENDED,
```

Comment:
this `record.status === FEE_RECORD_STATUS.TO_DO || record.status === FEE_RECORD_STATUS.TO_DO_AMENDED ` is also repeated in `dtfs-central-api/src/v1/validation/utilisation-report-service/fee-record-validator.ts`.  Could we make a helper for just the === ||| === and call that helper in both these places?

Reply:
Will add 👍